owners.json

```diff
@@ -18,5 +18,7 @@
   "urbanserj": {"slack": "urbanserj", "jira": "sergeyurbanovich"},
   "vinodkone": {"slack": "vinodkone", "jira": "vinod"},
   "Poltergeist": {"slack": "philipph", "jira": "philipph"},
-  "pierrebeitz": {"slack": "pierre", "jira": "pierrebeitz"}
+  "pierrebeitz": {"slack": "pierre", "jira": "pierrebeitz"},
+  "janisz": {"slack": "janisz", "jira": "tomaszjaniszewski"},
```

Comment:
```suggestion
```

I prefer to not be an owner. IMO the only owner should be @orsenthil. Most of our problems are result of previous overrides and merges without tests or lack of tests.

Reply:
Ownership will make you Github PR approval count for Mergebot as far as I know.